src/locale/locales/id/messages.po

```diff
@@ -0,0 +1,2478 @@
+
+#: src/view/com/modals/crop-image/CropImage.web.tsx:122
+msgid "Square"
+msgstr "Persegi"
+
+#: src/view/com/auth/create/Step1.tsx:90
+#: src/view/com/modals/ServerInput.tsx:62
+msgid "Staging"
+msgstr "Staging"
+
+#: src/view/screens/Settings.tsx:730
+msgid "Status page"
+msgstr "Halaman status"
+
+#: src/view/screens/Settings.tsx:666
+msgid "Storybook"
+msgstr "Storybook"
+
+#: src/view/com/modals/AppealLabel.tsx:101
+msgid "Submit"
+msgstr "Kirim"
+
+#: src/view/screens/ProfileList.tsx:574
+msgid "Subscribe"
+msgstr "Langganan"
+
+#: src/view/screens/ProfileList.tsx:570
+msgid "Subscribe to this list"
+msgstr "Langganan ke daftar ini"
+
+#: src/view/screens/Search/Search.tsx:349
+msgid "Suggested Follows"
+msgstr "Saran untuk Diikuti"
+
+#: src/view/screens/Support.tsx:30
+#: src/view/screens/Support.tsx:33
+msgid "Support"
+msgstr "Dukungan"
+
+#: src/view/com/modals/SwitchAccount.tsx:115
+msgid "Switch Account"
+msgstr "Pindah Akun"
+
+#: src/view/screens/Settings.tsx:646
+msgid "System log"
+msgstr "Log sistem"
+
+#: src/view/com/modals/crop-image/CropImage.web.tsx:112
+msgid "Tall"
+msgstr "Tinggi"
+
+#: src/view/shell/desktop/RightNav.tsx:85
+msgid "Terms"
+msgstr "Ketentuan"
+
+#: src/view/screens/Settings.tsx:744
+#: src/view/screens/TermsOfService.tsx:29
+#: src/view/shell/Drawer.tsx:256
+msgid "Terms of Service"
+msgstr "Ketentuan Layanan"
+
+#: src/view/com/modals/AppealLabel.tsx:70
+#: src/view/com/modals/report/InputIssueDetails.tsx:50
+msgid "Text input field"
+msgstr "Area input teks"
+
+#: src/view/com/profile/ProfileHeader.tsx:306
+msgid "The account will be able to interact with you after unblocking."
+msgstr "Akun ini akan dapat berinteraksi dengan Anda setelah blokir dibuka."
+
+#: src/view/screens/CommunityGuidelines.tsx:36
+msgid "The Community Guidelines have been moved to <0/>"
+msgstr "Panduan Komunitas telah dipindahkan ke <0/>"
+
+#: src/view/screens/CopyrightPolicy.tsx:33
+msgid "The Copyright Policy has been moved to <0/>"
+msgstr "Kebijakan Hak Cipta telah dipindahkan ke <0/>"
+
+#: src/view/com/post-thread/PostThread.tsx:433
+msgid "The post may have been deleted."
+msgstr "Post mungkin telah dihapus."
+
+#: src/view/screens/PrivacyPolicy.tsx:33
+msgid "The Privacy Policy has been moved to <0/>"
+msgstr "Kebijakan Privasi telah dipindahkan ke <0/>"
+
+#: src/view/screens/Support.tsx:36
+msgid "The support form has been moved. If you need help, please<0/> or visit {HELP_DESK_URL} to get in touch with us."
+msgstr "Formulir bantuan telah dipindahkan. Jika Anda butuh bantuan, silakan<0/> atau kunjungi {HELP_DESK_URL} untuk menghubungi kami."
+
+#: src/view/screens/TermsOfService.tsx:33
+msgid "The Terms of Service have been moved to"
+msgstr "Ketentuan Layanan telah dipindahkan ke"
+
+#: src/view/com/util/ErrorBoundary.tsx:35
+msgid "There was an unexpected issue in the application. Please let us know if this happened to you!"
+msgstr "Sepertinya ada masalah di aplikasi. Beri tahu kami jika Anda mengalaminya!"
```

Comment:
Suggesting a change to `Sepertinya ada masalah di dalam aplikasi. Silahkan beri tahu kami jika Anda mengalaminya!`

Reply:
Yes, I think thats a good suggestion. :-)